test/TestAsset/http-status-codes.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
> I believe this may be end up leading to a case of service misuse.

One http request a day will not trigger misuse :-)

 > One solution would be to add a pre-release script, something like this in composer.json ("scripts" block).
 > 
 > "pre-release-tasks": [
 > "@update-test-asset"
 > ],
 > "update-test-asset": "curl -o test/TestAsset/http-status-codes.xml https://www.iana.org/assignments/http-status-codes/http-status-codes.xml"
 > 
 > What do you think?

If it's not automated, it's not checked. Let's please push it to the test suite, and run it every time.